app/templates/components/orders/company-info.hbs

```diff
@@ -0,0 +1,106 @@
+<div class="ui segments">
+  <div class="ui green inverted segment center aligned">
+    <div class="ui inverted mini statistic horizontal">
+      <div class="value">
+        {{t 'Company Information'}}
+      </div>
+    </div>
+  </div>
+  <div class="ui padded segment">
+    <table class="ui very basic compact unstackable table company-info">
+      <tbody>
+        {{#if this.data.registeredCompany}}
```

Comment:
Won't work, this is classical syntax. Use octane syntax, change data prop to tax